treeherder/model/models.py

```diff
@@ -210,14 +210,14 @@ def __str__(self):
 
 @python_2_unicode_compatible
 class Bugscache(models.Model):
-    id = models.AutoField(primary_key=True)
-    status = models.CharField(max_length=64, blank=True, db_index=True)
+    id = models.PositiveIntegerField(primary_key=True)
+    status = models.CharField(max_length=64, db_index=True)
     resolution = models.CharField(max_length=64, blank=True, db_index=True)
     summary = models.CharField(max_length=255)
     crash_signature = models.TextField(blank=True)
     keywords = models.TextField(blank=True)
     os = models.CharField(max_length=64, blank=True)
-    modified = models.DateTimeField(null=True, blank=True)
+    modified = models.DateTimeField()
```

Comment:
This may be beyond the scope of this PR since you're just trying to get parity, but we should consider using the ``auto_now`` param for ``DateTimeField``: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.DateField.auto_now

Maybe in another PR.  I noticed we have a save method for the ``Job`` model that could use it.